bin/migrate-tips.py

```diff
@@ -4,9 +4,15 @@
 db = db(env())
 
 slugs = db.all("""
-    SELECT slug
-      FROM teams
-     WHERE is_approved IS TRUE
+    SELECT distinct(t.slug)
+      FROM teams t
+      JOIN tips ON t.owner = tips.tippee
```

Comment:
It'd be great if we could add comments here to explain what we're filtering based on. For example, 

<img width="789" alt="screen shot 2016-01-24 at 12 53 36 pm" src="https://cloud.githubusercontent.com/assets/3893573/12535006/8497b0e0-c299-11e5-9057-59e9b138bfe2.png">